src/CMakeLists.txt

```diff
@@ -189,6 +189,30 @@ if(ENABLE_CURSES)
 	endif()
 endif(ENABLE_CURSES)
 
+option(ENABLE_POSTGRESQL "Enable PostgreSQL backend" TRUE)
+set(USE_POSTGRESQL FALSE)
+
+if(ENABLE_POSTGRESQL)
+	find_program(PG_CONFIG_EXECUTABLE pg_config DOC "pg_config")
+	find_library(PG_LIBRARY pq)
+	if(PG_CONFIG_EXECUTABLE)
+		execute_process(COMMAND ${PG_CONFIG_EXECUTABLE} --includedir-server
+			OUTPUT_VARIABLE PostgreSQL_SERVER_INCLUDE_DIRS
+			OUTPUT_STRIP_TRAILING_WHITESPACE)
+		set(PostgreSQL_ADDITIONAL_SEARCH_PATHS ${PostgreSQL_SERVER_INCLUDE_DIRS})
+	endif()
+
+	find_package("PostgreSQL")
+
+	if(PG_LIBRARY AND PG_CONFIG_EXECUTABLE)
+		set(USE_POSTGRESQL TRUE)
+		message(STATUS "PostgreSQL backend enabled")
+		include_directories(${PostgreSQL_INCLUDE_DIR})
```

Comment:
Is there any reason why PostgreSQL_INCLUDE_DIR does not follow the convention of ALL_CAPS like the rest of the build variables?